### PR TITLE
Content area upload

### PR DESCRIPTION
**Pull request by H3ll3m4**
[Trello](https://trello.com/c/IXwHazop/3898-content-area-in-file-upload-component-of-presenter)

This adds a area section on the upload component. This simply render the body attribute of the page. The body was Body section and has been updated to empty string. It will defaults as Optional content. We also remove the body from the list of surplus property in the schemas as it is now used for the content area. In the Editor it looks like this:
image

### CODE REVIEW

**Review by StevenLeighton21 — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ All notable changes to this project will be documented in this file.
 ### Changed
 ### Fixed
 
+## [3.3.15] - 2024-02-08
```
Comment (zheileman) [inline CHANGELOG.md:10]: Date 😄

File: config/locales/cy.yml [hunk 2]
```diff
@@ -66,13 +66,14 @@ cy:
...
       multiupload:
+        upload_file: '(cy) Upload a file'
         remove_file: '(cy) Delete'
         add_another: '(cy) Add another file'
         single_upload: '(cy) Maximum file size 7MB'
         answered_singular: '(cy) You can add 1 more file (maximum 7MB per file)'
         answered_plural: '(cy) You can add %{num} more files (maximum 7MB per file)'
-        answered_count_singular: '(cy) You have added 1 file'
-        answered_count_plural: '(cy) You have added %{num} files'
+        answered_count_singular: '(cy) You have uploaded 1 file'
```
Comment (zheileman) [inline config/locales/cy.yml:75]: As we've changed some copy we will need a way to communicate these to the hmcts translation team. Maybe check with Darren?

**Review by zheileman — APPROVED** [review]
👍

**Pull request merged**